example/create_dependencies.py

```diff
@@ -74,6 +87,19 @@ def usage():
     # pylint: disable=invalid-name
     all_files = sys.argv[1:]
 
+    # Create a look-up directories that maps the filenames without
```

Comment:
"Create a mapping of filenames without path and extensions to the full filename"

Reply:
Done.